File: OOP/Homework/08JavaOOP/src/geometry/Circle.java
package geometry;

public class Circle extends PlaneShape{

	private double radius;

	public Circle(double x, double y, double radius) {
		super(x, y);
		
		this.radius = radius;
	}

	public double getRadius() {
		return radius;
	}

	public void setRadius(double radius) {
		this.radius = radius;
	}

	@Override
	public double CalculateArea() {
		return Math.PI * Math.pow(radius, 2);
	}

	@Override
	public double CalculatePerimeter() {
		return 2* Math.PI * this.radius;
	}

	@Override
	public String toString() {
		return "Circle [radius="
				+ radius
				+ ", "
				+ (vertices != null ? "vertices=" + vertices[0]
						: "") + "]";
	}
	
	
	
}

File: OOP/Homework/06EncapsulationPolymorphism/06.EncapsulationPolymorphism/Shapes/Circle.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Shapes
{
    class Circle:IShape
    {
        private double radius;

        public double Radius { get; set; }

        public Circle(double radius)
        {
            this.Radius = radius;
        }

        public double CalculateArea()
        {
            return Math.PI * Math.Pow(this.Radius, 2);
        }

        public double CalculatePerimeter()
        {
            return 2 * Math.PI * this.Radius;
        }
    }
}
